changelogs/fragments/2230-java_keystore-1669-ssl-input-files-by-path.yml

```
@@ -0,0 +1,10 @@
+---
+minor_changes:
+  - "java_keystore - add options ``certificate_path`` and ``private_key_path``,
+    mutually exclusive with ``certificate`` and ``private_key`` respectively, and
+    targetting files on remote hosts rather than their contents on the controller.
+    Refactor documentation: document new options, add info for better understanding
+    of what options fail and what options overwrite the keystore when their values
+    change, fix indentation, use present tenses. Refactor integration tests to run
+    the same tasks twice: one as before and one with the new options
```

Docs changes are only documented in case some important changes have been made (like serious bugfixes), and test changes are usually never mentioned in the changelog.
```suggestion
    targetting files on remote hosts rather than their contents on the controller
```

---

done